vogue/commands/base.py

```diff
@@ -23,11 +25,27 @@
 @click.version_option(__version__)
 @click.group(cls=FlaskGroup,
              create_app=create_app,
-             add_default_commands=True,
+             add_default_commands=True, 
              invoke_without_command=False,
              add_version_option=False)
-def cli(**_):
+@click.option("-c", "--config", type=click.File(), help="Path to config file")
+@click.option("-u", "--db-uri", type=str, default='mongodb://localhost:27030', help="If no config")
```

Comment:
I suggest making help a bit more descriptive:

```suggestion
@click.option("-u", "--db-uri", type=str, default='mongodb://localhost:27030', help="Set db uri if no config is provided")
```